Android/notifications/src/main/java/com/facebook/notifications/internal/view/HeroView.java

```diff
@@ -116,15 +116,17 @@ private static int getRoundedCorners(@NonNull CardConfiguration cardConfiguratio
   @Override
   protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
     LayoutParams params = (LayoutParams) assetView.getLayoutParams();
-    params.height = LayoutParams.WRAP_CONTENT;
+    // Params can return null if the assetView is not yet attached to the screen.
+    if (params != null) {
```

Comment:
Sure, doesn't matter much, the important part is that we need to explicitly call into super here.